src/main/java/com/iridium/iridiumskyblock/commands/PrivateCommand.java

```diff
@@ -0,0 +1,64 @@
+package com.iridium.iridiumskyblock.commands;
+
+import com.iridium.iridiumskyblock.IridiumSkyblock;
+import com.iridium.iridiumskyblock.api.IridiumSkyblockAPI;
+import com.iridium.iridiumskyblock.database.Island;
+import com.iridium.iridiumskyblock.database.User;
+import com.iridium.iridiumskyblock.utils.StringUtils;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Command which makes User's island to unvisitable'
+ */
+public class PrivateCommand extends Command {
+
+    /**
+     * The default constructor.
+     */
+    public PrivateCommand() {
+        super(Collections.singletonList("private"), "Make your island to unvisitable", "", true);
+    }
+
+    /**
+     * Executes the command for the specified {@link CommandSender} with the provided arguments.
+     * Not called when the command execution was invalid (no permission, no player or command disabled).
+     * Makes User's island to unvisitable
+     *
+     * @param sender The CommandSender which executes this command
+     * @param args   The arguments used with this command. They contain the sub-command
+     */
+    @Override
+    public void execute(CommandSender sender, String[] args) {
+        Player player = (Player) sender;
+        User user = IridiumSkyblockAPI.getInstance().getUser(player);
+        Optional<Island> island = user.getIsland();
+        if (island.isPresent()) {
+            island.get().setVisitable(false);
+            int amount = IridiumSkyblock.getInstance().getIslandManager().privateIsland(island.get());
+            player.sendMessage(StringUtils.color(IridiumSkyblock.getInstance().getMessages().islandNowPrivate
+                    .replace("%prefix%", IridiumSkyblock.getInstance().getConfiguration().prefix)
+                    .replace("%amount%", String.valueOf(amount))));
+        } else {
+            player.sendMessage(StringUtils.color(IridiumSkyblock.getInstance().getMessages().dontHaveIsland.replace("%prefix%", IridiumSkyblock.getInstance().getConfiguration().prefix)));
+        }
+    }
+
+    /**
+     * Handles tab-completion for this command.
+     *
+     * @param commandSender The CommandSender which tries to tab-complete
+     * @param command       The command
+     * @param label         The label of the command
+     * @param args          The arguments already provided by the sender
+     * @return The list of tab completions for this command
+     */
+    @Override
+    public List<String> onTabComplete(CommandSender commandSender, org.bukkit.command.Command command, String label, String[] args) {
+        return null;
```

Comment:
Don't return null, check the other commands

Reply:
This was my fault, I may have returned null in some of my commands.

Im not sure which way around it is but one of them will return a list of online players the other will return nothing

Reply:
Yeah, you usually just returned null, but I fixed it for you. A player list is just ugly, so Collections.emptyList() is the right choice.